docs/grammar/SolidityParser.g4

```diff
@@ -13,7 +13,6 @@ sourceUnit: (
 	pragmaDirective
 	| importDirective
 	| contractDefinition
-	| errorDefinition
```

Comment:
Is this change correct?

Reply:
It seems so. Apparently #11218 added a second `errorDefinition` here - not that there's another one at the end of this list. I think it must have been a bug so reverting it is what we want.